distutils/ccompiler.py

```diff
@@ -6,18 +6,20 @@
     get_default_compiler,
     new_compiler,
     show_compilers,
+    _default_compilers,
 )
 from .compilers.C.errors import CompileError, LinkError
 
 __all__ = [
-    'CompileError',
-    'LinkError',
-    'compiler_class',
-    'gen_lib_options',
-    'gen_preprocess_options',
-    'get_default_compiler',
-    'new_compiler',
-    'show_compilers',
+    "CompileError",
+    "LinkError",
+    "compiler_class",
+    "gen_lib_options",
+    "gen_preprocess_options",
+    "get_default_compiler",
+    "new_compiler",
+    "show_compilers",
+    _default_compilers,
```

Comment:
This needs to be a string.
```suggestion
    "_default_compilers",
```

Reply:
Yes it does, thanks! I was struggling getting the environment set up correctly and there were so many red and yellow lines I got distracted :(